app/components/map/index.js

```diff
@@ -480,22 +527,38 @@ class Map extends Component {
                 </MapView.Marker>
               : null
             }
-            {selectedAlertCoordinates &&
-              <MapView.Marker
-                key={'selectedAlert'}
-                coordinate={selectedAlertCoordinates}
-                image={alertWhite}
-                anchor={{ x: 0.5, y: 0.5 }}
-                zIndex={10}
-              />
+            {neighbours && neighbours.length > 0 &&
+              neighbours.map((neighbour, i) => (
+                <MapView.Marker
+                  key={i}
```

Comment:
I would namespace the key prop with something like `neighbours` because we have other `map` as a sibling, and therefore keys will be repeated probably.